test/test_actionsmap.py

```diff
@@ -266,7 +266,7 @@ def import_mock(name, globals={}, locals={}, fromlist=[], level=-1):
     with pytest.raises(MoulinetteError) as exception:
         amap.process({}, timeout=30, route=("GET", "/test-auth/none"))
 
-    expected_msg = "unable to load function % s.%s because: %s" % (
+    expected_msg = "unable to load function {: }.{} because: {}".format(
```

Comment:
Not sure to understand why but that one is making tests crash :thinking: 

```
           expected_msg = "unable to load function {: }.{} because: {}".format(
              "moulitest",
              "testauth_none",
  >           "Yoloswag",
          )
  E       ValueError: Sign not allowed in string format specifier
```

Reply:
Ah, maybe the ":" in "{: }"? But that looks weird :/

Reply:
Is there any reason to have a space between `%` and `s`? We could just replace `{: }` by `{}`